### ISSUE TITLE
Icon request: icon-xml

### ISSUE THREAD

**Issue by Berkmann18** [ForkAwesome/Fork-Awesome#99]
XML is (alongside JSON) the core language when it comes to data interchange and configurations and more too (e.g: Windows gadget development, Android development, Semantic Web, ...).

**Comment by xuv** [comment]
Agreed, XML is an important format. But does it have a logo? Not sure about that.

**Comment by Berkmann18** [comment]
There are several variants of it, not sure what most people around the world would see as the official one although the XML-file or <xml> ones seems more recognised.

**Comment by Berkmann18** [comment]
@xuv Is there more info needed for this request?

**Comment by xuv** [comment]
Yes. As they say on Wikipedia, [Reference Needed] on general use of an "official" logo for XML.

I've done some research and, although there are many versions of very different XML logos, there is nothing that stands out as universal acceptance of what that symbol would be (in terms of shape, font face, arrangements and other decorations that go with it) 

So if anyone could make a case, with references, for what could be defined as an official symbol for XML, then I feel  we could fulfill this request.

**Comment by Berkmann18** [comment]
Okay, I'll try to more research on that and see.
